lib/generate.js

```diff
@@ -15,46 +15,44 @@ const fetchResource = require('./fetchResource');
  * @param {Object} options - Options Object
  * @param {string|Array} options.algorithms - Desired hash algorithm(s)
  * @param {string} options.url - Resource URL
- * @return {Promise}
+ * @return {Object}
```

Comment:
it still returns a Promise, see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function#Return_value

Reply:
fixed.